docs/source/guides/contributor/chapters/environment.rst

```diff
@@ -20,6 +40,9 @@ Or if you already have pip installed, you can run directly::
 
  $ pip install -r requirements-dev.txt
 
+If you intend to build the documentation location, please also run::
```

Comment:
Didn't you mean locally instead of location?

Reply:
Yes, I did mean "locally".  Thanks for catching that!  I've sent a version with that typo fixed.